person.rb

```diff
@@ -26,4 +27,8 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def add_rentals(rent)
+    @rentals.push(rent)
+  end
```

Comment:
- [ ] In The add_rentals method, you have to create a rental passing all the required parameters. Here, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(book, date)
  Rental.new(date, self, book)
end
```